### PR TITLE
[8.0.0][CDE] ApiTableRowPrToJSON check

### PR DESCRIPTION
**Pull request by Yar04ek**

### CODE REVIEW

File: js/docx/smoke/api_table_row_pr/to_json.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
+    let oTableStyle = oDocument.CreateStyle("CustomTableStyle", "table");
+        oTableStyle.SetName("My Custom Table Style");
+    let oTableRowPr = oTableStyle.GetTableRowPr();
+        oTableRowPr.SetHeight("atLeast", 720);
+    let jsonTableStyle = oTableStyle.ToJSON();
```
Comment (askonev) [inline js/docx/smoke/api_table_row_pr/to_json.js:8]: You test TableRowPr.ToJSON() but convert TableStyle

File: spec/docx/smoke/api_table_row_properties_spec.rb [hunk 2]
```diff
@@ -16,4 +16,17 @@
     docx = builder.build_and_parse('js/docx/smoke/api_table_row_pr/set_table_header.js')
     expect(docx.elements[1].properties.table_style.table_row_properties.table_header).to be_truthy
   end
+
+  it 'ApiTableRowPr | ToJson method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_table_row_pr/to_json.js')
+    json = docx.elements[1].nonempty_runs.map(&:text).join
+    parsed_json = JSON.parse(json)
+    expect(parsed_json).to include('name' => 'My Custom Table Style')
+    expect(parsed_json['tblPr']).to include('tblOverlap' => 'never', 'inline' => false, 'type' => 'tablePr')
```
Comment (askonev) [inline spec/docx/smoke/api_table_row_properties_spec.rb:25]: I think it's bad practice to use `include` everywhere. It is better to use the full path to the field where the required information is located

File: js/docx/smoke/api_table_row_pr/to_json.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        GlobalVariable["JSON_TableStyle"] = jsonTableStyle;
+        oParagraph.AddText(jsonTableStyle);
+builder.CloseFile();
+
+//////////////
+
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
```
Comment (askonev) [inline js/docx/smoke/api_table_row_pr/to_json.js:17]: For what purpose do you refer to this element?

**Review by askonev — CHANGES_REQUESTED** [review]

File: js/docx/smoke/api_table_row_pr/to_json.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+let jsonTableStyle = GlobalVariable["JSON_TableStyle"];
+let oTableStyle = Api.FromJSON(jsonTableStyle);
+let oTable = Api.CreateTable(3, 3);
+oTable.SetStyle(oTableStyle);
+let jsonTable = oTableStyle.ToJSON();
+let oParagraph2 = Api.CreateParagraph();
+oParagraph2.AddText(jsonTable);
+oDocument.Push(oParagraph2);
+builder.SaveFile("docx", "TableStyleToJSON.docx");
+builder.CloseFile();
```
[the remark this answers is not in the document]
Comment (Yar04ek) [reply js/docx/smoke/api_table_row_pr/to_json.js]: Fix in the new commit.

File: js/docx/smoke/api_table_row_pr/to_json.js (hunk 1 above)
Comment (Yar04ek) [reply js/docx/smoke/api_table_row_pr/to_json.js]: Fix in the new commit.

File: spec/docx/smoke/api_table_row_properties_spec.rb (hunk 2 above)
Comment (Yar04ek) [reply spec/docx/smoke/api_table_row_properties_spec.rb]: Fix in the new commit.

File: js/docx/smoke/api_table_row_pr/to_json.js (hunk 3 above)
Comment (Yar04ek) [reply js/docx/smoke/api_table_row_pr/to_json.js:16]: Fix in the new commit.

**Comment by askonev** [comment]
`dublicate`

**Pull request closed without merging**